Assets/AnkrSDK/Scripts/Data/SupportedWallets.cs

```diff
@@ -0,0 +1,46 @@
+using System.Linq;
+
+namespace AnkrSDK.Data
+{
+	public static class SupportedWallets
+	{
+		public static Wallet[] WebGL =
+		{
+			Wallet.None,
+			Wallet.Metamask,
+			Wallet.Torus
+		};
+		public static Wallet[] Mobile =
+		{
+			Wallet.None,
+			Wallet.Metamask,
+			Wallet.Trust
+		};
+		public static Wallet[] Standalone =
+		{
+			Wallet.None,
+			Wallet.Metamask,
+			Wallet.Trust
+		};
+
+		public static string[] GetWebGLWallets()
+		{
+			return WalletsToArray(WebGL);
+		}
+		
+		public static string[] GetMobileWallets()
```

Comment:
these are not used?

Reply:
Yes, but it can be used if we will do the same wallet managment in other platforms.